src/fparser/one/tests/test_do_block_r814.py

```diff
@@ -48,43 +52,43 @@
 @pytest.mark.parametrize('name', [None, 'loop_name'])
 @pytest.mark.parametrize('label', [None, '123'])
 @pytest.mark.parametrize('control_comma', [False, True])
-@pytest.mark.parametrize('initial_expression',
-                         ['1', '10', 'x+y', 'size(array)', 'size(this%array)'])
 @pytest.mark.parametrize('terminal_expression',
                          ['1', '10', 'x+y', 'size(array)', 'size(this%array)'])
-@pytest.mark.parametrize('incrament_expression',
-                         ['1', '10', 'x+y', 'size(array)', 'size(this%array)'])
 @pytest.mark.parametrize('end_name', [None, 'loop_name', 'wrong_name'])
 @pytest.mark.parametrize('end_label', [None, '123', '456'])
-def test_do(name, label, control_comma, initial_expression,
-            terminal_expression, incrament_expression, end_name, end_label):
+def test_do(name, label, control_comma, terminal_expression,
+            end_name, end_label):
     # pylint: disable=redefined-outer-name, too-many-arguments, too-many-locals
     '''
-    Checks that the "do" loop parser understands all forms of the syntax.
+    Checks that the "do" loop parser understands the "for-next" variant of
+    the syntax. This is defined in BS ISO/IEC 1539-1:2010 with R814-R822.
+
+    TODO: Only the terminal expression is tested. This is a short-cut and
+          relies on expression handling being applied identically across
+          all expressions. This was true at the time of writing the test.
     '''
     name_snippet = name + ': ' if name else None
     label_snippet = label + ' ' if label else None
     comma_snippet = ', ' if control_comma else None
-    inc_snippet = ', ' + incrament_expression if incrament_expression else None
-    do_code = '''{name}do {label}{comma}variable = {init}, {term}{inc}
+    # TODO: Although the Fortran standard allows for "continue" to be used in
```

Comment:
Please just change this to comment to say that fparser1 doesn't support it and fparser1 is being deprecated. (Otherwise we have to have an issue associated with the TODO and nobody wants that...)